packages/scripts/src/testDeterminism.ts

```diff
@@ -58,7 +61,7 @@ async function hashTables(sequelize: Sequelize, tables: string[]): Promise<Table
 
   const hashes: TableHashes = new Map();
   for (const table of tables) {
-    await sequelize.query(`CREATE TEMPORARY TABLE determinism_check_table AS TABLE "${table}"`);
+    await sequelize.query(`CREATE TEMPORARY TABLE determinism_check_table AS TABLE ${table}`);
```

Comment:
Is removing the double quotes here fixing the issue? In general I've felt using double quotes to be safer than not in postgres

Reply:
Oh wait now I read the code properly I see it gets wrapped earlier, all good :+1: